pkg/scheduler/flavorassigner/flavorassigner.go

```diff
@@ -109,8 +109,13 @@ func (a *Assignment) ToAPI() []kueue.PodSetAssignment {
 func (a *Assignment) TotalRequestsFor(wl *workload.Info) resources.FlavorResourceQuantities {
```

Comment:
Please add a comment indicating that in the case of `PartialAdmission`, we scale the returned `Requests`.